solutions/logging/core-experimentation/setters.yaml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+######
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: setters
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  #############
+  # General Settings Values
+  #
+  # The following are Settings for the environment to bootstrap with
+  #
+  #############
+  billing-id: "0000000000"
+  org-id: "0000000000"
+  #############
+  # Logging Project ID
+  management-project-id: management-project-12345
+  logging-prj-id: logging-prj-id-12345
+  #############
+  # LoggingLogBucket retention settings
+  # Set the number of days to retain logs in Cloud Logging buckets
+  # Set the lock mechanism on the bucket to: true or false
+  # After a retention policy is locked (true), you can't delete the bucket until every log in the bucket has fulfilled the bucket's retention period
+  # AU-9 PROTECTION OF AUDIT INFORMATION
+  # AU-11 AUDIT RECORD RETENTION
+  retention-locking-policy: false
```

Comment:
same as earlier comment, should we pre-set defaults?

Reply:
If you wanted to test out an LZ deploy or just this package for example and set these values `retention-locking-policy: true` and `retention-in-days: 365`  you could end up locking your testing environment for 365 days. As soon as logs would be written into the logs buckets, you would need to wait for 365 days for the data to expire before being able to delete the logging project.

Reply:
Because there should not be any Production data or impacts to the business, retention periods can be justifiably reduced from the standard.  We can discuss how long is reasonable if needed, unless to Luc's example the variable will change if just testing LZ deploy, that will be blown away in a couple days.  What impact would it have to set to 30 by default? or 29 days if the bill cycle is 30? :-)

Reply:
Every example I've seen has used a false setting for the locking policy and 1 day retention period. It's much easier to turn on locking and modify the retention period after the fact than the other way around, which I believe is actually impossible. When locking is on, a bucket can only be deleted once the data inside the bucket has expired per the number of retention days.

I don't think the decision on these settings should be made at this level anyways. We should hash out our policies and apply them at the tier1 repo source-customization folder in AzDO.